src/overlays/actors/ovl_Bg_Bdan_Switch/z_bg_bdan_switch.c

```diff
@@ -59,17 +59,31 @@ extern CollisionHeader D_06005CF8;
 extern Gfx D_060061A0[];
 extern Gfx D_06005A20[];
 
-static ColliderJntSphItemInit sJntSphItemsInit[] = {
+static ColliderJntSphElementInit sJntSphElementsInit[] = {
     {
-        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0xEFC1FFFE, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
-        { 0x00, { { 0x0000, 0x0078, 0x0000 }, 370 }, 100 },
+        {
+            ELEMTYPE_UNK0,
+            { 0x00000000, 0x00, 0x00 },
+            { 0xEFC1FFFE, 0x00, 0x00 },
+            TOUCH_OFF,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { 0, { { 0, 120, 0 }, 370 }, 100 },
     },
 };
 
 static ColliderJntSphInit sJntSphInit = {
-    { COLTYPE_UNK10, 0x00, 0x09, 0x39, 0x20, COLSHAPE_JNTSPH },
+    {
+        COLTYPE_NONE,
+        AT_OFF,
+        AC_ON | AC_PLAYER,
+        OC_ON | OC_ALL,
+        OT_TYPE2,
+        COLSHAPE_JNTSPH,
+    },
     1,
-    sJntSphItemsInit,
+    sJntSphElementsInit,
```

Comment:
It doesn't have to happen in this set of changes, but I think it could be good to design macros for collider inits, so we can reorder arguments as we want, and most importantly avoid duplicated information such as the `COLSHAPE` and the `ARRAY_COUNT` in colliders that can have multiple elements (ie. the 1 right before the pointer to the init array in this example)

Reply:
I've been considering this as well. Definitely something for later, but it will be a good thing to have.